pkg/skaffold/survey/config_test.go

```diff
@@ -0,0 +1,182 @@
+/*
+Copyright 2021 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package survey
+
+import (
+	"testing"
+	"time"
+
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/util"
+	"github.com/GoogleContainerTools/skaffold/testutil"
+)
+
+func TestSurveyPrompt(t *testing.T) {
+	tests := []struct {
+		description string
+		s           config
+		expected    string
+	}{
+		{
+			description: "hats survey",
+			s:           hats,
+			expected: `Help improve Skaffold with our 2-minute anonymous survey: run 'skaffold survey'
+`,
+		},
+		{
+			description: "not hats survey",
+			s: config{
+				id:         "foo",
+				promptText: "Looks like you are using foo feature. Help improve Skaffold foo feature and take this survey",
+				expiresAt:  time.Date(2021, time.August, 14, 00, 00, 00, 0, time.UTC),
```

Comment:
Wondering if we should include this field since it's not actually used.  Maybe set it in the past to catch and ensure `prompt()` doesn't use the `expiresAt`.
```suggestion
				expiresAt:  time.Date(1999, time.August, 14, 00, 00, 00, 0, time.UTC),
```